docs/upm_readme.md

```diff
@@ -26,13 +26,14 @@ The follow target platforms are supported in Unity for the current release of th
 | Universal Windows Platform x64 | Supported |
 | Android | Supported |
 | iOS | Supported |
-| Linux | [Preview](docs/linux/linux_supported_versions.md) |
-| MacOS | [Preview](docs/macOS/macOS_supported_versions.md) |
+| Linux | Ubuntu 18.04<sup>(#linux-support)</sup> |
```

Comment:
Is the (#linux-support) rendering correctly? I'd expect it to be a superscript "1" to line up with the note on the bottom. Are they supposed to link to each other? Because that doesn't work if so. 

![image](https://github.com/PlayEveryWare/eos_plugin_for_unity/assets/50968607/a88b05bf-c249-4371-8381-5ddb3490e586)

Reply:
You are correct - that is not rendering correctly. I'll get that fixed.